<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftProjectorClass</class>
 <widget class="QMainWindow" name="SoftProjectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="softprojector.qrc">
    <normaloff>:/icons/icons/Main.ico</normaloff>:/icons/icons/Main.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Tab">
        <attribute name="title">
         <string>Tab</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2"/>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="clear_button">
          <property name="toolTip">
           <string>Stop displaying text to the screen (display black screen or wallpaper). Sortcut for this button is the Escape key.</string>
          </property>
          <property name="text">
           <string>Hide (Esc)</string>
          </property>
          <property name="shortcut">
           <string>Esc</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="show_button">
          <property name="toolTip">
           <string>If Clear was pressed earlier, this will re-display the text to the screen</string>
          </property>
          <property name="text">
           <string>Show (F4)</string>
          </property>
          <property name="shortcut">
           <string>F4</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="labelShow">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QListWidget" name="listShow">
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionNewSong"/>
    <addaction name="actionEditSong"/>
    <addaction name="actionDeleteSong"/>
    <addaction name="separator"/>
    <addaction name="actionManage_Database"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="action_Help"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/settings.ico</normaloff>:/icons/icons/settings.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/base-22x22-actions-application-exit.png</normaloff>:/icons/icons/base-22x22-actions-application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEditSong">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/song_edit.png</normaloff>:/icons/icons/song_edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit Current Song...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionNewSong">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/song_new.png</normaloff>:/icons/icons/song_new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Song...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionManage_Database">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/database.png</normaloff>:/icons/icons/database.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage Database...</string>
   </property>
   <property name="toolTip">
    <string>Import and export song collections and Bibles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionDeleteSong">
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/song_delete.png</normaloff>:/icons/icons/song_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Current Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/lang_ru.png</normaloff>
     <selectedoff>:/icons/icons/lang_ru_ck.png</selectedoff>:/icons/icons/lang_ru.png</iconset>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="softprojector.qrc">
     <normaloff>:/icons/icons/lang_en.png</normaloff>:/icons/icons/lang_en.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
